data/known_issue/onprem-ent.json

```
@@ -1,296 +1,86 @@
 [
     {
-        "spec_file": "tests/integration/accessibility/accessibility_post_spec.js",
+        "spec_file": "tests/integration/channel/new_channel_with_board_spec.js",
         "cases": [
             {
-                "title": "Verify Accessibility Support in Post > MM-T1486_3 Verify Tab support on Post on Center Channel",
+                "title": "New Channel modal with Boards enabled > MM-T5141 New Channel is created with an associated Board",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```

For now, let's have the type either flaky and bug. Other could be test infra/server limitation but not sure what's the short term for it.
If this is flaky, then:
```suggestion
                "type": "flaky"
```